### DISCUSSION TITLE
Branch "Applied Machine and Deep Learning (ML)" and "Natural Language Processing (NLP)" Information

### DISCUSSION THREAD

**Discussion by Viddesh1** [tusuii/All-Pracs#15] [Q&A]
Hello 👋  @tusuii ,

I was checking 📜 out the below branches of Semester - 2 in the Main branch of the remote repository:-

Semester - II:
Applied Machine and Deep Learning (ML). [View](https://github.com/tusuii/All-Pracs/tree/Practical-Course-onApplied-Machine-and-Deep-Learning/ML%20and%20DL%20practicals)
Natural Language Processing (NLP). [View](https://github.com/tusuii/All-Pracs/tree/NLP/NLP)

--------------------------------------------------

I see that the this branch name is too long for this subject find below:-
Applied Machine and Deep Learning (ML). [View](https://github.com/tusuii/All-Pracs/tree/Practical-Course-onApplied-Machine-and-Deep-Learning/ML%20and%20DL%20practicals)

Rather than such a long name what about a shorter name to the branch something like "ML" etc.

What do you think regarding the branch name for above branch? 🤔 

Also, The same branch "Applied Machine and Deep Learning (ML). [View](https://github.com/tusuii/All-Pracs/tree/Practical-Course-onApplied-Machine-and-Deep-Learning/ML%20and%20DL%20practicals)" 🤔. Is the "ML and DL practicals" directory is relevant for the subject of  "Applied Machine and Deep Learning (ML)" ? Because, I see there are many files with .jl extentions and "AIP_Pract_ALL" directory. So, Which folder and files are relevant for this subject "Applied Machine and Deep Learning (ML)"? 🤔 

--------------------------------------------------

The "Natural Language Processing (NLP). [View](https://github.com/tusuii/All-Pracs/tree/NLP/NLP)" branch looks good to me. 

Thanks!

**Comment by tusuii** [comment]
yes will change branch name for sure :100:

**Comment by tusuii** [comment]
Thanks for the recommendation @Viddesh1 done renameing the long name
